hub/compression.py

```diff
@@ -36,10 +40,7 @@
     c for c in IMAGE_COMPRESSIONS if c.upper() in Image.SAVE and c.upper() in Image.OPEN
```

Comment:
What is the maximum line width that we are using? If it is 79 then I think this line is too long